docs/content/en/docs/core-components/api-manual.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Manual API Description
+description: >
+    Descriptions for the SOARCA manual interaction REST API endpoints 
+categories: [API]
+tags: [protocol, http, rest, api]
+weight: 3
+date: 2024-05-21
+---
+
+## Endpoint descriptions
+
+We will use HTTP status codes https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
+
+
+```plantuml
+@startuml
+protocol Reporter {
+    GET     /manual
+    POST    /manual/continue
+}
+@enduml
+```
+
+
+### /manual
+The manual interaction endpoint for SOARCA
+
+#### GET `/manual`
+Get all pending manual actions objects that are currently waiting in SOARCA.
+
+##### Call payload
+None
+
+##### Response
+200/OK with payload:
+
+
+
+|field              |content                |type               | description |
```

Comment:
This is a list of manual command action steps right? The return payload as presented is only for one such steps? I see that in the example it's indeed a list, but maybe we should mention it here in this response bit too